frontend/public/components/monitoring/receiver-forms/email-receiver-form.tsx

```diff
@@ -0,0 +1,297 @@
+import * as _ from 'lodash-es';
+import * as React from 'react';
+
+import { SectionHeading } from '../../utils';
+import { SaveAsDefaultCheckbox } from './alert-manager-receiver-forms';
+
+const SMTP_GLOBAL_FIELDS = [
+  'from',
+  'smarthost',
+  'hello',
+  'auth_username',
+  'auth_password',
+  'auth_identity',
+  'auth_secret',
+  'require_tls',
+];
+const GLOBAL_FIELDS = [...SMTP_GLOBAL_FIELDS]; //TODO follow up PR will add advanced fields
+
+// Ex: converts 'auth_password' to 'emailAuthPassword'
```

Comment:
Why do we need to make this conversion? Can't we use the same name as in the Alertmanager config?

Reply:
This is for the form field names, ex: `<input name=emailAuthPassword' ...>`

Reply:
Tracking in another comment